social-network/src/Summary/Summary.js

```diff
@@ -0,0 +1,28 @@
+import React,{Component} from 'react';
```

Comment:
I'm not sure whether there's enough functionality and re-usability there to justify a separate component for Summary and Image

Reply:
I kind of went down the rabbit hole on this one.  I just kept breaking things down until I couldn't anymore.  >.<  Lesson learned.  =]